CHANGES_UNRELEASED.md

```diff
@@ -3,11 +3,23 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.53.2...master)
+
+## Rust
+
+### What's New
+
+- There is no need to `source libs/bootstrap-desktop.sh` anymore! ([#2769](https://github.com/mozilla/application-services/pull/2769))
+    - Please install the `sqlcipher` library system-wide so it can be picked up by the build system.
+        - Linux: `apt-get install -y sqlcipher`
```

Comment:
Ditto here -- This is too old in ubuntu (and AFAICT missing from other apt repositories).